src/specs/models/experiment-bodies/ProcessingConfig.v2.yaml

```diff
@@ -0,0 +1,28 @@
+title: Processing Config v2
+description: Schema for data processing configuration
+type: object
+properties:
+  cellSizeDistribution:
+    $ref: ../processing-config-bodies/ProcessingConfigCellSizeDistribution.v1.yaml
+  mitochondrialContent:
+    $ref: ../processing-config-bodies/ProcessingConfigMitochondrialContent.v1.yaml
+  classifier:
+    $ref: ../processing-config-bodies/ProcessingConfigClassifier.v1.yaml
+  numGenesVsNumUmis:
+    $ref: ../processing-config-bodies/ProcessingConfigNumGenesVsNumUMIs.v1.yaml
+  doubletScores:
+    $ref: ../processing-config-bodies/ProcessingConfigDoubletScores.v1.yaml
+  dataIntegration:
+    $ref: ../processing-config-bodies/ProcessingConfigDataIntegration.v1.yaml
+  configureEmbedding:
+    $ref: ../processing-config-bodies/ProcessingConfigConfigureEmbedding.v1.yaml
+  meta:
+    $ref: ../processing-config-bodies/ProcessingConfigMeta.v1.yaml
+required:
```

Comment:
can you add `additionalProperties: false`?

Reply:
to avoid the ui passing fields that we don't support

Reply:
Before doing this we will need to clean up the processingconfigs in dynamo (or after migrating) because they have a lot of stuff.

Perhaps I should perform that cleanup in the dynamoToSql migration script?

Reply:
as discussed on a call, we can make the schema stricter as part of separate piece of work